storage/uri_test.go

```diff
@@ -52,23 +53,37 @@ func TestURI_Parent(t *testing.T) {
 
 	parent, err := storage.Parent(storage.NewURI("file:///foo/bar/baz"))
 	assert.Nil(t, err)
-	assert.Equal(t, "file:///foo/bar/", parent.String())
```

Comment:
The comment above notes the importance of trailing slashes, but this alters that behaviour...
I would tend to agree with the above and that a fileURI.Parent() would always end in a '/'.

Reply:
I changed it so that ```Parent()``` always returns a path with a trailing ```/```